[MECA2755] Add exam January 2019 (#671)
Gp2mv3/Syntheses: src/q7/autoindu-MECA2755/exam/2019/Janvier/All/autoindu-MECA2755-exam-2019-Janvier-All.tex

\documentclass[en]{../../../../../../eplexam}

\usepackage{float}
\usepackage{../../../../../../eplunits}

\hypertitle{Industrial automation}{7}{MECA}{2755}{2019}{Janvier}{All}
{Martin Braquet \and Louis Fichefet}
{Paul Fisette, Bruno Dehez and Renaud Ronsse}

\section{Robotics part}

\textit{The statements for this part should be available on the EPL-Drive or on Moodle.}

\subsection{Robot}

The robot is a microsurgical robot where the the wrist is assumed to be spherical.
Give:
\begin{itemize}

>     \item the number of freedom degrees

Reviewer: ```suggestion
    \item the number of degrees of freedom
```

    \item the shape of its workspace
    \item the direct and inverse kinematic

>     \item the jacobian and the singular positions.

Reviewer: ```suggestion
    \item the Jacobian and the singular positions.
```

\end{itemize}

\subsection{PD Controller}

> In order to control the end effector motion in the task space, make the block diagram of the PD controller and explain the utility of the jacobian.

Reviewer: ```suggestion
In order to control the end effector motion in the task space, make the block diagram of the PD controller and explain the utility of the Jacobian.
```

\subsection{Via Points}

Explain the different methods and give the advantages and drawbacks for each of them.

\nosolution

\section{Pneumatic part}

\subsection{Theory I}

What is the purpose of this mechanism? How is it called? Explain its working (use the letters in the figure to reference the different parts).

\begin{figure}[H]
    \centering
    \includegraphics{regulator.png}
\end{figure}

\subsection{Cycle}

> A pneumatic cycle is shown in Figure \ref{cycle}. It is asked to:

Reviewer: ```suggestion
A pneumatic cycle is shown in \figuref{cycle}. It is asked to:
```

\begin{itemize}
    \item give the name of this cycle (square, L, U, \dots)
    \item give its sequence in one line
    \item differentiate the monostable and bistable components of the circuit.
\end{itemize}
Then:
\begin{itemize}
    \item give the motion diagram
    \item write all the functions ($f(\dots) = \dots$)
    \item \textit{simply and clearly} write this cycle in a GRAFCET program.
\end{itemize}

\begin{figure}[H]
    \centering
    \includegraphics{Ucycle.png}
    \caption{Pneumatic cycle}

>     \label{cycle}

Reviewer: ```suggestion
    \label{fig:cycle}
```
So that `\figuref` works ;)

\end{figure}

\subsection{Theory II}

> What does represent these symbols? Insert your answer in the right column of the table (be concise).

Reviewer: ```suggestion
What do these symbols represent? Insert your answer in the right column of the table (be concise).
```
